backend/src/main/java/wooteco/retrospective/infrastructure/dao/member/MemberDao.java

```diff
@@ -43,13 +43,17 @@ public Member insert(Member member) {
     public Optional<Member> findById(Long id) {
         String query = "SELECT id, name FROM MEMBER WHERE id = ?";
 
-        return Optional.ofNullable(this.jdbcTemplate.queryForObject(query, ROW_MAPPER, id));
+        return this.jdbcTemplate.query(query, ROW_MAPPER, id)
+                .stream()
+                .findAny();
```

Comment:
이런식의 해결은 임시방편이라고 생각합니다. 에러가 발생했던 이유가 EmptyResultDataAccessException 이었던 만큼. 그 에러를 핸들링 했어야 하지 않을까요?

```suggestion
        try {
            return Optional.ofNullable(
                    jdbcTemplate.queryForObject(query, ROW_MAPPER, id)
            );
        } catch (EmptyResultDataAccessException e) {
            return Optional.empty();
        }
```

Reply:
너잘이 제시한 코드에서 `Optional.ofNullable()`을 사용했으므로 try-catch 구문은 필요 없지 않을까요?

Reply:
너잘 덕분에 하나 알아갑니다 👍

Reply:
@da-nyee 아까 슬랙에서 말했던 이유로 try-catch가 필요해용 ㅠㅠ

Reply:
나도 해당 서제스쳔에 동의~보감~